### ISSUE TITLE
Deleting a transition causes saving exception 'null'

### ISSUE THREAD

**Issue by yawlfoundation** [yawlfoundation/yawl#543]
```
What steps will reproduce the problem?
1. Open the attached file 'last correct.yawl' in the editor.
2. In the bottom row, delete the leftmost transition having an OR-split.
3. Save the file (see attached 'wrong.yawl').
4. Close and reopen the saved file.
5. In the bottom row, add an arc between the leftmost transition and the first 
transition with an OR-join.
6. Try to save the file.

What is the expected output? What do you see instead?
Expect save to succeed. Instead, an error message 'null' is given and the model 
is not saved.

What version of the product are you using? On what operating system?
Yawl editor 3.0.1 build 564. Windows 7/8.1

Please provide any additional information below.
I guess the incoming arcs of the OR-join are not cleared correctly when 
deleting the transition.

```

Original issue reported on code.google.com by `sander.l...@gmail.com` on 20 Mar 2015 at 12:42

Attachments:
* [wrong.yawl](https://storage.googleapis.com/google-code-attachments/yawl/issue-543/comment-0/wrong.yawl)
* [last correct .yawl](https://storage.googleapis.com/google-code-attachments/yawl/issue-543/comment-0/last correct .yawl)

**Comment by yawlfoundation** [comment]
```
I was unable to replicate this issue (after many attempts). Can you please 
check again with the latest build and advise how it goes?
```

Original comment by `yawl.mic...@gmail.com` on 23 Mar 2015 at 3:06

**Comment by yawlfoundation** [comment]
Original comment by `yawl.mic...@gmail.com` on 24 Mar 2015 at 12:21
* Changed state: **UnderObservation**

**Comment by yawlfoundation** [comment]
```
Updated to build 569.

Issue remains. Steps to reproduce:
- open wrong.yawl
- add arrow from the leftmost transition in the lower branch to the OR-join 
closest to it
- attempt to save
```

Original comment by `sander.l...@gmail.com` on 25 Mar 2015 at 2:11

Attachments:
* [Untitled.mp4](https://storage.googleapis.com/google-code-attachments/yawl/issue-543/comment-3/Untitled.mp4)

**Comment by yawlfoundation** [comment]
```
There is a problem in updating the editor, so that when the update completes 
and the editor restarts itself, on Windows it is still reloading the old jar 
file (3.0) rather than the new one (3.0.1). You can tell which of the 2 
versions are actually running by looking on the title bar of the editor window. 
Only the new jar file actually contains the updates.

If you run the new jar file, you'll find this issue to no longer be a problem.

Updating from 3.0 to 3.0.1 also 'breaks' the start menu item's properties. 
These can be edited manually to the new jar's name if desired. Work continues 
on fixing the menu items programmatically.

Please confirm the running YAWLEditor3.0.1.jar fixes the issue of saving 
'wrong.yawl'.
```

Original comment by `yawl.mic...@gmail.com` on 31 Mar 2015 at 7:21

**Comment by yawlfoundation** [comment]
```
Fixed in 3.0.1
```

Original comment by `yawl.mic...@gmail.com` on 1 May 2015 at 12:59
* Changed state: **Fixed**
